strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout-provider/src/test/java/org/carlspring/strongbox/services/ArtifactManagementServiceImplTest.java

```diff
@@ -164,36 +165,46 @@ public void initialize()
                                       "7.3"  // Used by testArtifactResolutionFromGroup()
         );
 
-        Repository repositoryGroup = mavenRepositoryFactory.createRepository(STORAGE0, REPOSITORY_GROUP);
+        MutableRepository repositoryGroup = mavenRepositoryFactory.createRepository(REPOSITORY_GROUP);
         repositoryGroup.setType(RepositoryTypeEnum.GROUP.getType());
         repositoryGroup.setAllowsRedeployment(false);
         repositoryGroup.setAllowsDelete(false);
         repositoryGroup.setAllowsForceDeletion(false);
         repositoryGroup.addRepositoryToGroup(REPOSITORY_RELEASES);
 
-        createRepositoryWithArtifacts(repositoryGroup,
+        createRepositoryWithArtifacts(STORAGE0,
+                                      repositoryGroup,
                                       "org.carlspring.strongbox:strongbox-utils",
                                       "8.2" // Used by testDeploymentRedeploymentAndDeletionAgainstGroupRepository()
         );
 
         // Used by testForceDelete()
-        Repository repositoryWithTrash = mavenRepositoryFactory.createRepository(STORAGE0,
-                                                                                 REPOSITORY_RELEASES_WITH_TRASH);
+        MutableRepository repositoryWithTrash = mavenRepositoryFactory.createRepository(REPOSITORY_RELEASES_WITH_TRASH);
         repositoryWithTrash.setTrashEnabled(true);
 
-        createRepositoryWithArtifacts(repositoryWithTrash,
+        createRepositoryWithArtifacts(STORAGE0,
+                                      repositoryWithTrash,
                                       "org.carlspring.strongbox:strongbox-utils",
                                       "7.2");
 
         // Used by testRemoveTimestampedSnapshots()
-        Repository repositorySnapshots = mavenRepositoryFactory.createRepository(STORAGE0, REPOSITORY_SNAPSHOTS);
+        MutableRepository repositorySnapshots = mavenRepositoryFactory.createRepository(REPOSITORY_SNAPSHOTS);
         repositorySnapshots.setPolicy(RepositoryPolicyEnum.SNAPSHOT.getPolicy());
 
-        createRepository(repositorySnapshots);
+        createRepository(repositorySnapshots, STORAGE0);
```

Comment:
Same as my earier comment -- `storageId` first and then the `repositoryId`. Let's keep this consistent, because someday somebody will expect that the parameters are like this across the codebase and will make the mistake of passing in the parameters incorrectly.